nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -1047,6 +1047,49 @@ def test_forward_rel_ops():
     _test_forward_rel_op([t1, t2], math_ops.equal)
     _test_forward_rel_op([t1, t2], math_ops.not_equal)
 
+#######################################################################
```

Comment:
Suggest to add TOPI level test cases with all targets enabled.

Reply:
@srkreddy1238 added new tests. Please take a look.